### ISSUE TITLE
training time issue

### ISSUE THREAD

**Issue by imian00** [hizhangp/yolo_tensorflow#6]
Hi I'm a beginner in this deep-learning field and now I'm trying to fine-tune some of the well-known object detectors such as Yolo. first I want to train Yolo to classify a single class(person or none) and also to find bounding boxes of multiple objects(people) in an image or in videos

I think I can use just the PASCAL VOC data sets for fine-tuning. and I have a laptop with i5-6200U, Geforce 940M. I know its not such a good environment and condition especially for training CNNs.

so.. how do you expect the duration time for fine-tuning this Yolo, if I use roughly 1000 images? 

thanks in advance

p.s 
sorry for my bad english and i hope you understood my intention!

**Comment by hizhangp** [comment]
@imian00 Duration time mainly depends on the size of your data and validation loss. In general, you can decrease the learning rate by a factor of 10 after every 2\~3 epochs, and stop iteration when validation loss does not decreases (usually about 5\~6 epochs).

**Comment by imian00** [comment]
thanks for your kind reply :)

I've never heard of the term 'validation' so I searched that term in google and read the concept.

Is it like testing the model during training operation with validation data sets which were never shown to the model before? and is it to find optimum weights and biases which is the intermediate state between under-fitted state(loss still decreasing) and over-fitted state(loss start increasing) right? 

Then could you tell me how often do I have to do the validation during the training session?
such as every 1 epoch or 0.5 epoch et cetra.

by the way, Xie Xie and Thank you so much for your great job! this project is not as giant as other yolo based projects and only have necessary parts for YOLO, redundancy-free.  so i can learn basic concepts from it more easily!

**Comment by hizhangp** [comment]
Yes, that is validation.

Doing validation every 0.25 epoch may be OK. Just make sure that training loss and validation convergence at the same time.

Hoping this will be helpful for you :)

**Comment by imian00** [comment]
Thank you! It helped me a lot!
祝你万事亨通 bro!
